Signal/src/ViewControllers/InboxTableViewCell.m

```diff
@@ -227,16 +227,8 @@ - (void)configureWithThread:(TSThread *)thread
         self.unreadLabel.hidden = YES;
     }
 
-    NSString *threadIdCopy = thread.uniqueId;
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
-        UIImage *avatar =
-            [OWSAvatarBuilder buildImageForThread:thread contactsManager:contactsManager diameter:kavatarViewDiameter];
-        dispatch_async(dispatch_get_main_queue(), ^{
-            if ([_threadId isEqualToString:threadIdCopy]) {
-                self.avatarView.image = avatar;
-            }
-        });
-    });
+    self.avatarView.image =
```

Comment:
Specifically, I'm assuming that perf is a function of input avatar size, so maybe we should modify the "create fake contacts" logic to include a randomly generated image, whose file size/content sizes vary.